Dockerfile

```diff
@@ -1,24 +1,32 @@
-FROM puckel/docker-airflow:1.10.9
-USER root
+FROM apache/airflow:2.1.2-python3.8
 ENV POETRY_VIRTUALENVS_CREATE=false \
     POETRY_CACHE_DIR='/var/cache/pypoetry' \
     GOOGLE_APPLICATION_CREDENTIALS='/usr/local/airflow/service-account.json'
 
+USER root
 RUN apt-get update \
     && apt-get install -y --no-install-recommends git \
-    # 1. if you don't need postgres, remember to remove postgresql-dev and sqlalchemy
-    # 2. libglib2.0-0 libsm6 libxext6 libxrender-dev libgl1-mesa-dev are required by opencv
-    # 3. git is required by pip install git+https
-    && pip install --no-cache-dir poetry==1.0.5 \
```

Comment:
poetry 1.0.5 is a bit old.

Reply:
I want to delete the duplicate comment (first one) https://github.com/pycontw/pycon-etl/pull/49#pullrequestreview-735507346 but I failed....